### ISSUE TITLE
Patterns

### ISSUE THREAD

**Issue by paulrobertlloyd** [k-next/plugins#44]
It would be great if the existing [patterns plugin](https://github.com/getkirby-plugins/patterns-plugin) could be ported to v3. Is this planned?

**Comment by jenstornell** [comment]
I made a similar plugin for Kirby 2 that now har a MIT license.

https://github.com/jenstornell/kirby-component-kit

**Comment by jenstornell** [comment]
**Kirby Components 0.1 alpha**

https://github.com/jenstornell/kirby-components

It's a rewrite of [Kirby Component Kit](https://github.com/jenstornell/kirby-component-kit). After using components I no longer want to make a site without it.

**Short info:**

- There is no UI. I never used the UI I created last time so I will spend no time on a UI this time.
- Blueprints does not yet work. https://github.com/k-next/kirby/issues/890
- When I've tested to build a new site with it, I will remove the `alpha` label.
- Template controllers is prefixed with `--`.

```text
site/components/
  --about/
    component.php
    controller.php
  --home/
  --projects/
    menu1/
    menu2/
  header/
  footer/
    component.php
    image.jpg
    style.scss
```

In conclusion, this is a much more lightweight plugin than my previous ones.
